docs/configuring.md

```diff
@@ -57,17 +57,54 @@ Keep in mind that this will override any existing file content.
 
 ## Pre-commit setup
 
-To use Ansible-lint with [pre-commit], add the following to the
+To use Ansible-lint with [pre-commit] tool, add the following to the
 `.pre-commit-config.yaml` file in your local repository.
 
+Do not confuse [pre-commit] tool with git hook feature with the same name.
+While [pre-commit] tool can also make use of git hooks, it does not require
```

Comment:
```suggestion
While [pre-commit] can also make use of git hooks, it does not require
```

Reply:
In the first few instances it's probably better to say "the [pre-commit] tool" and then just "[pre-commit]" in subsequent mentions.